diff --git a/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetBackground/BottomSheetBackground.js b/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetBackground/BottomSheetBackground.js
index 9ad453f..4dc1bdb 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetBackground/BottomSheetBackground.js
+++ b/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetBackground/BottomSheetBackground.js
@@ -15,9 +15,7 @@ const BottomSheetBackgroundComponent = ({
   style
 }) => /*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNative.View, {
   pointerEvents: pointerEvents,
-  accessible: true,
-  accessibilityRole: "adjustable",
-  accessibilityLabel: "Bottom Sheet",
+  accessible: false,
   style: [_styles.styles.container, style]
 });
 const BottomSheetBackground = /*#__PURE__*/(0, _react.memo)(BottomSheetBackgroundComponent);
diff --git a/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetBackground/BottomSheetBackground.js b/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetBackground/BottomSheetBackground.js
index 0e33e6a..3e0caa9 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetBackground/BottomSheetBackground.js
+++ b/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetBackground/BottomSheetBackground.js
@@ -9,9 +9,7 @@ const BottomSheetBackgroundComponent = ({
   style
 }) => /*#__PURE__*/_jsx(View, {
   pointerEvents: pointerEvents,
-  accessible: true,
-  accessibilityRole: "adjustable",
-  accessibilityLabel: "Bottom Sheet",
+  accessible: false,
   style: [styles.container, style]
 });
 const BottomSheetBackground = /*#__PURE__*/memo(BottomSheetBackgroundComponent);
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
index a6f048e..baffb64 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
@@ -1910,7 +1910,7 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
               detached={detached}
               style={_providedContainerStyle}
             >
-              <Animated.View style={containerStyle}>
+              <Animated.View style={containerStyle} nativeID={'BottomSheetComponent'}>
                 <BottomSheetBackgroundContainer
                   key="BottomSheetBackgroundContainer"
                   animatedIndex={animatedIndex}
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx
index 3f165dd..7536f01 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx
@@ -9,9 +9,7 @@ const BottomSheetBackgroundComponent = ({
 }: BottomSheetBackgroundProps) => (
   <View
     pointerEvents={pointerEvents}
-    accessible={true}
-    accessibilityRole="adjustable"
-    accessibilityLabel="Bottom Sheet"
+    accessible={false}
     style={[styles.container, style]}
   />
 );
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx
index 0716ecd..081d723 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx
@@ -30,10 +30,10 @@ function BottomSheetContainerComponent({
       {
         top: topInset,
         bottom: bottomInset,
-        overflow: detached ? 'visible' : 'hidden',
+        overflow: 'visible',
       },
     ],
-    [style, detached, topInset, bottomInset]
+    [style, topInset, bottomInset]
   );
   //#endregion
 
