diff --git a/node_modules/detox/src/android/matchers/native.js b/node_modules/detox/src/android/matchers/native.js
index 48615fd..96bb51b 100644
--- a/node_modules/detox/src/android/matchers/native.js
+++ b/node_modules/detox/src/android/matchers/native.js
@@ -57,7 +57,7 @@ class TypeMatcher extends NativeMatcher {
 }

 class VisibleMatcher extends NativeMatcher {
-  constructor(pct = 75) {
+  constructor(pct = 50) {
     super();

     if (pct !== undefined && (!Number.isSafeInteger(pct) || pct < 1 || pct > 100)) {
